src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -9,12 +10,16 @@ public class ShopService {
     /*** This class is used for creating new e-shop object and initializing it with some products.
      */
 
+
     public Eshop createShop() {
         /** Should create new e-shop object and initialize it with some products.
          * Should return e-shop object.
          */
-       ArrayList<Product> products = createProducts();
-        throw new UnsupportedOperationException("createShop() is not implemented yet");
+
+        ArrayList<Product> products = createProducts();
+        Eshop eshop = new Eshop("Fantazijos", products);
```

Comment:
You could inline this one since you just need to pass products to a constructor
`        Eshop eshop = new Eshop("Fantazijos", createProducts());
`